### PR TITLE
Hunt - SecEvent - Updating entities and putting in YAML format

### PR DESCRIPTION
**Pull request by shainw**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Hunting Queries/SecurityEvent/powershell_newencodedscipts.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  InitiatingProcessFileName=ParentProcessName,InitiatingProcessCommandLine="",InitiatingProcessParentFileName="";
+  processEvents};
+  let encodedPSScripts = 
+  ProcessCreationEvents 
+  | where TimeGenerated >= ago(midtime)
+  | where FileName =~ "powershell.exe"
+  | where ProcessCommandLine contains "-encodedCommand";
+  encodedPSScripts
+  | where TimeGenerated > ago(endtime)
+  | summarize count() by ProcessCommandLine
```
Comment (ms-bnick) [inline Hunting Queries/SecurityEvent/powershell_newencodedscipts.yaml]: Add a min(TimeGenerated) as a timestamp?
```suggestion
  | summarize count(), timestamp = min(TimeGenerated) by ProcessCommandLine
```

File: Hunting Queries/SecurityEvent/new_processes.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  processEvents};
+  ProcessCreationEvents
+  | where TimeGenerated >= ago(starttime) and TimeGenerated < ago(endtime)
+  | summarize HostCount=dcount(ComputerName) by tostring(FileName)
+  | join kind=rightanti (
+      ProcessCreationEvents
+      | where TimeGenerated >= ago(endtime)
+      | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), Computers = makeset(ComputerName) , HostCount=dcount(ComputerName) by tostring(FileName)
+  ) on FileName
+  | project StartTimeUtc, Computers, HostCount, FileName
```
[the remark this answers is not in the document]
Comment (shainw) [reply Hunting Queries/SecurityEvent/new_processes.yaml]: Done, thanks for catching.

File: Hunting Queries/SecurityEvent/powershell_downloads.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  | where TimeGenerated >= ago(timeframe) 
+  | where FileName in~ ("powershell.exe", "powershell_ise.exe")
+  | where ProcessCommandLine has "Net.WebClient"
+     or ProcessCommandLine has "DownloadFile"
+     or ProcessCommandLine has "Invoke-WebRequest"
+     or ProcessCommandLine has "Invoke-Shellcode"
+     or ProcessCommandLine contains "http:"
+  | project TimeGenerated, ComputerName, InitiatingProcessFileName, FileName, ProcessCommandLine
+  | top 100 by TimeGenerated
+  | extend timestamp = TimeGenerated, HostCustomEntity = ComputerName
```
[the remark this answers is not in the document]
Comment (shainw) [reply Hunting Queries/SecurityEvent/powershell_downloads.yaml]: Done, thanks for catching.

File: Hunting Queries/SecurityEvent/powershell_newencodedscipts.yaml (hunk 1 above)
Comment (shainw) [reply Hunting Queries/SecurityEvent/powershell_newencodedscipts.yaml]: Done, thanks for catching.

File: Hunting Queries/SecurityEvent/powershell_downloads.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  processEvents};
+  ProcessCreationEvents
+  | where TimeGenerated >= ago(timeframe) 
+  | where FileName in~ ("powershell.exe", "powershell_ise.exe")
+  | where ProcessCommandLine has "Net.WebClient"
+     or ProcessCommandLine has "DownloadFile"
+     or ProcessCommandLine has "Invoke-WebRequest"
+     or ProcessCommandLine has "Invoke-Shellcode"
+     or ProcessCommandLine contains "http:"
+  | project TimeGenerated, ComputerName, InitiatingProcessFileName, FileName, ProcessCommandLine
```
[the remark this answers is not in the document]
Comment (shainw) [reply Hunting Queries/SecurityEvent/powershell_downloads.yaml]: Done, thanks for catching.

**Pull request merged**